src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -45,7 +45,16 @@ public class RachisAdminHandler : RequestHandler
                 HttpContext.Response.Headers["Reached-Leader"] = "true";
 
                 var commandJson = await context.ReadForMemoryAsync(RequestBodyStream(), "external/rachis/command");
-                var command = CommandBase.CreateFrom(commandJson);
+                CommandBase command;
+                try
+                {
+                    command = CommandBase.CreateFrom(commandJson);
+                }
+                catch (InvalidOperationException e)
```

Comment:
What can cause this?

Reply:
When the deserializer for the command doesn't exists.

Reply:
More concrete example is when a 4.1 follower sends 4.1 command to his 4.0 leader.